scripts/azureml-assets/azureml/assets/release/model_publish_utils/mlflow_utils.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""MLFlow Operations Class."""
+
+
+class MLFlowModelUtils:
```

Comment:
Could we remove this file since there does not seem to be any functionality?

Reply:
It is a failure case for type mlflow_model in model.yaml